addons/website_sale_delivery/models/sale_order.py

```diff
@@ -11,6 +11,7 @@ class SaleOrder(models.Model):
         compute='_compute_amount_delivery',
         string='Delivery Amount',
         help="The amount without tax.", store=True, tracking=True)
+    access_point_address = fields.Text('Delivery Point Address')
```

Comment:
It should probably be a JSON field instead of text. It would avoid all the encode/decode works